### ISSUE TITLE
Ship info

### ISSUE THREAD

**Issue by 0x6C77** [TeslasCat/SpaceWars#8]
`/info ${ship}`

Display information about specified ship
- [ ] Name
- [ ] Owner
- [ ] Specs
- [ ] Trajectory

**Comment by PMaynard** [comment]
Would you say this is complete?

**Comment by 0x6C77** [comment]
Nope, the information isn't coming from the server. Plus I think the UI is broken in the current build and will be re-written.

**Comment by PMaynard** [comment]
Visually I don't see any issue with the UI.
I've updated issue #9 to reflect this.

**Comment by 0x6C77** [comment]
Seems to be working on macak, must be my local build. Still missing a few bits of info
